components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/JWTUtils.java

```diff
@@ -492,8 +492,13 @@ public static X509Certificate resolveSignerCertificate(IdentityProvider idp) thr
         X509Certificate x509Certificate;
         String tenantDomain = getTenantDomain();
         try {
```

Comment:
Refactor and move this to OAuth2Util

Reply:
fixed by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2758/commits/1f8cc44645bf831b6826250610bd014696842865